tests/functional/export_v2/cram/metadata-columns.t

```diff
@@ -0,0 +1,16 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+Run export with tree and metadata with additional columns.
+
+  $ ${AUGUR} export v2 \
+  >  --tree "$TESTDIR/../data/tree.nwk" \
+  >  --metadata "$TESTDIR/../data/dataset1_metadata_with_additional_columns.tsv" \
+  >  --metadata-columns "field_A" "field_B" \
+  >  --maintainers "Nextstrain Team" \
+  >  --output dataset.json > /dev/null
+
+  $ python3 "$TESTDIR/../../../../scripts/diff_jsons.py" "$TESTDIR/../data/dataset-with-additional-metadata-columns.json" dataset.json \
+  >   --exclude-paths "root['meta']['updated']" "root['meta']['maintainers']"
+  {}
```

Comment:
Suggestion: Test the feature more directly. Example:

```
Output has field_A and field_B.

  $ jq '.tree.children[0].node_attrs.field_A.value' dataset.json
  "AA"
  $ jq '.tree.children[0].node_attrs.field_B.value' dataset.json
  "AAA"
```

Reply:
We don't require `jq` in dev environments currently, but I think it would be appropriate to add if it enables a better testing experience.

Reply:
Totally agree! I'll defer those changes for now, will follow up later in https://github.com/nextstrain/augur/issues/1399.